index.d.ts

```diff
@@ -0,0 +1,82 @@
+import { Action, StoreEnhancer, Store } from "redux";
+
+export interface StoreCreator {
+  <S, T>(
+    reducer: EnhancedReducer<S>,
+    preloadedState: S,
+    enhancer: StoreEnhancer<S>
+  ): Store<S>;
+}
+
+export interface EnhancedReducer<S> {
+  <A extends Action>(state: S, action: A): S | [S, CmdType<A, Function>];
```

Comment:
maybe we can extract loop into its own type. People will need that

Reply:
not sure i'm understanding but the loop function is declared on line 57 

https://github.com/willisplummer/redux-loop/blob/96eb5a6ecc7921f9c1b9f2f1d16a68f0dc77152e/index.d.ts#L57

Reply:
im talking about the type of the return value of loop(), which is `[S, CmdType]`. That's returned by loop(), and also by loopreducers

Reply:
got it - added that